### ISSUE TITLE
Streamer badge disappearing, streams not appearing on main page

### ISSUE THREAD

**Issue by WandererXII** [WandererXII/lishogi#316]
Discussed on discord: https://discord.com/channels/778633701541806081/778633953992376350/842377835079401542

**Comment by justremember** [comment]
Streams also disappear from homepage & tournament page after a while of streaming.

**Comment by justremember** [comment]
[mituhiko](https://lishogi.org/streamer/mituhiko) and [axera](https://lishogi.org/streamer/Axera)'s streams not appearing on homepage. https://twitter.com/mituhiko25/status/1398752353025597444 https://twitter.com/MoE_Axera/status/1398786112403869701

**Comment by WandererXII** [comment]
We started hitting youtube api quota limit, I will try to get them to increase it. So that could be the reason why youtube streamers disappear after some time. Not sure what could cause Gotanda_N's problem though.

**Comment by justremember** [comment]
About Gotanda_N, nevermind, she's showing up in https://lishogi.org/streamer now. Lets see if it will show up in the homepage.

**Comment by WandererXII** [comment]
> About Gotanda_N, nevermind, she's showing up in https://lishogi.org/streamer now. Lets see if it will show up in the homepage. Her next shogi stream is on June 5, Sat 5:30-7:30 UTC if you are available to help debug. https://www.twitch.tv/gotanda_n/schedule

That's a bit too early for me, sorry. One thing that could've caused her not showing up, could be that she was streaming right after the twitch token expired. I try to check the logs every evening, but there still was a period when it wasn't working. I will prob merge this https://github.com/ornicar/lila/commit/9bc23ede2ed6530ba80e778bff2e9d53a7947f4a.
Currently I can't think of a reason why she wouldn't show up on the homepage, when other streamers do show up. Hopefully it works.


> The youtube api quota problem is weird, does the # of requests scale up w/ the # of streamers? If so, that's a problem...

Nope, I was just moving/cleaning some things between accounts/projects and didn't realize how much quota we really need. Currently waiting on google quota limit increase approval.

**Comment by justremember** [comment]
Apparently lichess is having trouble with youtube api too. I can't link the conversation right now cause of discord mobile app, but it is in their #support channel.

**Comment by Marken-Foo** [comment]
Lichess discord server, #support channel, relevant message link: https://discord.com/channels/280713822073913354/527867866700054538/852000515604611073

Open issue in lichess GitHub: https://github.com/ornicar/lila/issues/7316

**Comment by WandererXII** [comment]
Youtube approved our quota limit increase earlier today, so we should now be good on that front.

**Comment by ddugovic** [comment]
How do I livestream from Twitch with lishogi.org in my stream title and have Lishogi detect that I am streaming?  
![image](https://user-images.githubusercontent.com/1834123/178156139-a1cdef90-f9df-4174-ab49-788fe606bf6a.png)

I connected to Lishogi, set my livestream category, then started my stream, then failed this puzzle https://lishogi.org/training/Dmzmo

**Comment by ddugovic** [comment]
Jien advises that Twitch deleted the old "shogi" game category and added a new one.  I haven't yet looked up its Game ID (which might change again, who knows?).
https://dev.twitch.tv/docs/api/reference#get-games

**Comment by WandererXII** [comment]
Should be good now, if not please open a new issue.
